### PR TITLE
add test container ftp

### PR DESCRIPTION
**Pull request by Yuhu-kth**
gonna add comments later

### CODE REVIEW

**Comment by Yuhu-kth** [comment]
Screenshot 2024-08-04 at 19 28 58
I tested all the containers with tag0.6.0, it turns out 23 out of 56 are not running after started

**Comment by Yuhu-kth** [comment]
Screenshot 2024-08-06 at 21 34 08
I tried to assign "15.15.15.15" to container1, and use container2 to ping. Container2 was assigned to "15.15.15.2" automatically. The ping is unsuccessful, checked the network is connected to container1 correctly and container1 is running properly, checked container2 was connected to the same network too. Don't know why.
Tried to assign "15.15.15.2" to container1, then it can be pinged, like this
Screenshot 2024-08-06 at 22 21 38
I made the error to see the output, here you can see packets are received.
 but it's weird that container2 was still assigned to "15.15.15.2", I thought it will be assigned to another ip-address. I wanna ask your idea about it, because maybe its because I'm not familiar with network. I'll check tmr anyways.

File: emulation-system/tests/test.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+docker network create --subnet=15.15.15.0/24 my_custom_network
```
Comment (Limmen) [inline emulation-system/tests/test.sh:1]: Do we need this file?

**Review by Limmen — APPROVED** [review]
Looks good in general, but do we need the bash file? otherwise remove it and then I will merge

File: emulation-system/tests/test.sh (hunk 1 above)
Comment (Yuhu-kth) [reply emulation-system/tests/test.sh]: deleted

**Pull request merged**